### PR TITLE
feat/food countdown

### PR DESCRIPTION
**Pull request by boompttn**

### CODE REVIEW

File: pages/user/order/index.tsx [hunk 1]
```diff
@@ -33,19 +35,37 @@ const Orders: React.FC = observer(() => {
...
+          "Please select your menu before time limit:"]}
+        childrenClassName="flex justify-center "
+        className="m-4"
+      >
+        <Countdown format />
+      </Card>
+
+
+
+      {/* <p>We have a lot of food for you to choose! We have partnered wuth restaurants on floor 4 and 5, and food stalls are available on floor 8.</p>
```
Comment (new5558) [inline pages/user/order/index.tsx]: remove comments

File: pages/user/order/index.tsx [hunk 2]
```diff
@@ -33,19 +35,37 @@ const Orders: React.FC = observer(() => {
...
 
   return (
-    <div>
-      <div className='text-sm'>
+    <div className="bg-red-200">
+      <div className='text-sm '>
         Ordering as{' '}
         <span className='font-bold'>{userStore.userInfo?.username}</span>
       </div>
+      <Card
```
Comment (new5558) [inline pages/user/order/index.tsx]: just use card component on my #28 . There is no need to use 'items' and 'childrenClassName' as props of card component.

File: pages/user/order/index.tsx [hunk 3]
```diff
@@ -33,19 +35,37 @@ const Orders: React.FC = observer(() => {
...
           <OrderItem onOrder={orderFood} order={order} />
+
         </div>
       );
     });
 
   return (
-    <div>
-      <div className='text-sm'>
+    <div className="bg-red-200">
```
Comment (new5558) [inline pages/user/order/index.tsx]: bg-red-200 is unsupported due to #27. please change to others

File: pages/user/order/index.tsx [hunk 4]
```diff
@@ -33,19 +35,37 @@ const Orders: React.FC = observer(() => {
       return (
         <div key={order.key} className='my-3'>
           <OrderItem onOrder={orderFood} order={order} />
+
```
Comment (new5558) [inline pages/user/order/index.tsx]: remove this space. Or just use prettier to auto format your code.

File: pages/user/order/countDown.tsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  const minutes = (d2.getMinutes() - d1.getMinutes()) % 60;
+  const seconds = (d2.getSeconds() - d1.getSeconds()) % 60;
+  const res: Date = new Date();
+  res.setHours(hours);
+  res.setMinutes(minutes);
+  res.setSeconds(seconds);
+  return res;
+};
+
+function padZero(n: string, width: number) {
```
Comment (new5558) [inline pages/user/order/countDown.tsx:18]: I think addZeroPadding should be better name for this function, since padZero is not a verb.

**Review by new5558 — CHANGES_REQUESTED** [review]

File: pages/user/order/countDown.tsx (hunk 5 above)
Comment (fivechjr) [reply pages/user/order/countDown.tsx:18]: But “pad” is a verb: https://www.merriam-webster.com/dictionary/pad 🤔

Comment (new5558) [reply pages/user/order/countDown.tsx:18]: I see

File: report.20200121.134328.18473.0.001.json [hunk 6]
```diff
@@ -0,0 +1,703 @@
+
```
Comment (new5558) [inline report.20200121.134328.18473.0.001.json]: Remove this.

**Review by new5558 — CHANGES_REQUESTED** [review]

**Review by new5558 — APPROVED** [review]

**Pull request merged**